spark/src/main/java/org/apache/iceberg/spark/SparkSchemaUtil.java

```diff
@@ -70,17 +70,18 @@ public static Schema schemaForTable(SparkSession spark, String name) {
    *
    * @param spark a Spark session
    * @param name a table name and (optional) database
-   * @return a PartitionSpec for the table, if found
+   * @return a PartitionSpec for the table
    * @throws AnalysisException if thrown by the Spark catalog
    */
   public static PartitionSpec specForTable(SparkSession spark, String name) throws AnalysisException {
```

Comment:
It seems that this method is not even being used. Can we just delete it?

Reply:
I'm probably going to use it soon. I'd like to build a Hive to Iceberg table import utility that would use this.

Reply:
OK. Also a minor point but, do you think this change was more suitable in the `identitySpec` method which is defined as 
```
 private static PartitionSpec identitySpec(Schema schema, List<String> partitionNames) {
    if (partitionNames == null || partitionNames.isEmpty()) {
      return null;
    }
```
?